nutkit/protocol/error_type.py

```diff
@@ -97,40 +98,37 @@ class ErrorType(Enum):
     # in direct mode python throws service unavailable
     # dotnet and java throw read timeout, in java it extends service
     # unavailable
-    CONNECTION_READ_TIMEOUT_ERROR = "connection_read_timeout_error"
+    CONNECTION_READ_TIMEOUT_ERROR = "ConnectionReadTimeoutError"
 
     # TO BE DISCUSSED
     # direct
     # python throws when connection drops after commit
     # java throws service unavailable error
     # there is a test that expects retry to be interrupted, java and dotnet
     # keep trying
-    INCOMPLETE_COMMIT_ERROR = "incomplete_commit_error"
+    INCOMPLETE_COMMIT_ERROR = "IncompleteCommitError"
 
     # TO BE DISCUSSED
     # direct
     # go throws this
     # others seems to throw service unavailable
-    CONNECTIVITY_ERROR = "connectivity_error"
+    CONNECTIVITY_ERROR = "ConnectivityError"
 
     # direct
     # - when result has been consumed
-    RESULT_CONSUMED_ERROR = "result_consumed_error"
+    RESULT_CONSUMED_ERROR = "ResultConsumedError"
 
     # configuration
     # - illegal argument value
-    ILLEGAL_ARGUMENT_ERROR = "illegal_argument_error"
+    ILLEGAL_ARGUMENT_ERROR = "IllegalArgumentError"
 
     # protocol violations
-    PROTOCOL_ERROR = "protocol_error"
+    PROTOCOL_ERROR = "ProtocolError"
 
     # TO BE DISCUSSED
     # python throws when transaction has been closed
     # java throws client exception
-    TRANSACTION_ERROR = "transaction_error"
-
-    # TO BE DISCUSSED
-    MANAGED_TRANSACTION_ERROR = "managed_transaction_error"
```

Comment:
I decided to ditch `MANAGED_TRANSACTION_ERROR = "managed_transaction_error"` as this was only used to signify to the user that they can't manage (e.g., `.commit`) a tx inside a tx function. Since we decided to re-design this and won't/can't have tests for the new approach, there is no sense in having this error class any longer.

Reply:
Yes, this makes sense 👍